src/node_options.cc

```diff
@@ -327,10 +327,7 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::experimental_policy_integrity,
             kAllowedInEnvironment);
   Implies("--policy-integrity", "[has_policy_integrity_string]");
-  AddOption("--experimental-repl-await",
-            "experimental await keyword support in REPL",
-            &EnvironmentOptions::experimental_repl_await,
-            kAllowedInEnvironment);
```

Comment:
The option should be left in for now as a non-op to keep existing uses from breaking.

Reply:
`AddOption("--experimental-repl-await"", "experimental await keyword support in REPL",NoOp{}, kAllowedInEnvironment);` should be fine?

Reply:
Yes

Reply:
Done, thanks.